spec/signable/envelope_spec.rb

```diff
@@ -0,0 +1,39 @@
+require "spec_helper"
+
+describe Signable::Envelope do
+  describe '#update' do
+    it 'raises an error' do
+      expect { described_class.new.update }.to raise_error('not available')
+    end
+  end
+
+  describe '#delete' do
+    it 'raises an error' do
+      expect { described_class.new.delete }.to raise_error('not available')
+    end
+  end
+
+  describe '#cancel' do
+    it 'calls cancel on the Client object passing envelopes endpoint and fingerprint column value' do
+      object = described_class.new
```

Comment:
I think this should work, no need for the stubing.

```suggestion
      described_class.new(fingerprint: "FINGERPRINT_COLUMN_VALUE")
```

Reply:
👍